docs/references/javascript/types/sign-in-initial-values.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: SignInInitialValues
+description: The type for the `initialValues` prop that is used to pre-populate the sign-in form.
+---
+
+# `SignInInitialValues`
+
+The type for the `initialValues` prop that is used to pre-populate the sign-in form.
+
+## Attributes
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `emailAddress?` | `string` | The email address of the user. |
+| `username?` | `string` | The username of the user. |
+| `phoneNumber?` | `string` | The phone number of the user. |
```

Comment:
Should these have `/| undefined` like `SignUpInitialValues` has?

```suggestion
| `emailAddress?` | `string /| undefined` | The email address of the user. |
| `username?` | `string /| undefined` | The username of the user. |
| `phoneNumber?` | `string /| undefined` | The phone number of the user. |
```

Reply:
Yes, they can be `undefined`.